mkdocs.yml

```diff
@@ -2,82 +2,82 @@ site_name: Fiware-DataModels
+      - OnStreetParking      : Parking/OnStreetParking/doc/spec.md
+      - ParkingGroup         : Parking/ParkingGroup/doc/spec.md
+      - ParkingAccess        : Parking/ParkingAccess/doc/spec.md 
+      - ParkingSpot          : Parking/ParkingSpot/doc/spec.md
+  - Parks & Gardens:
+      - Introduction         : ParksAndGardens/doc/introduction.md
+      - Garden               : ParksAndGardens/Garden/doc/spec.md
+      - GreenspaceRecord     : ParksAndGardens/GreenspaceRecord/doc/spec.md
+      - FlowerBed            : ParksAndGardens/FlowerBed/doc/spec.md
+  - Points of Interest:
+      - Introduction         : PointOfInterest/doc/introduction.md
+      - PointOfInterest      : PointOfInterest/PointOfInterest/doc/spec.md
+      - Beach                : PointOfInterest/Beach/doc/spec.md
+      - Museum               : PointOfInterest/Museum/doc/spec.md
+      - WeatherStation       : PointOfInterest/WeatherStation/doc/spec.md
+      - AirQualityStation    : PointOfInterest/AirQualityStation/doc/spec.md
+  -  Point Of Interacion:
+      - SmartPointOfInteraction : PointOfInteraction/SmartPointOfInteraction/doc/spec.md
+      - SmartSpot               : PointOfInteraction/SmartSpot/doc/spec.md
+  -  Street Lighting:
+      - Introduction              : StreetLighting/doc/introduction.md
+      - Streetlight               : StreetLighting/Streetlight/doc/spec.md
+      - StreetlightModel          : StreetLighting/StreetlightModel/doc/spec.md
+      - StreetlightGroup          : StreetLighting/StreetlightGroup/doc/spec.md
+      - StreetlightControlCabinet : StreetLighting/StreetlightControlCabinet/doc/spec.md
+  - Transportation:
+      - Introduction           : Transportation/doc/introduction.md
+      - BikeHireDockingStation : Transportation/Bike/BikeHireDockingStation/doc/spec.md
+      - Road                   : Transportation/Road/doc/spec.md
+      - RoadSegment            : Transportation/RoadSegment/doc/spec.md
+      - TrafficFlowObserved    : Transportation/TrafficFlowObserved/doc/spec.md
+      - Vehicle                : Transportation/Vehicle/Vehicle/doc/spec.md
+      - VehicleModel           : Transportation/Vehicle/VehicleModel/doc/spec.md
+  -  Weather:
+      - Introduction         : Weather/doc/introduction.md
+      - WeatherObserved      : Weather/WeatherObserved/doc/spec.md
+      - WeatherForecast      : Weather/WeatherForecast/doc/spec.md
+      - WeatherAlert         : Weather/WeatherAlert/doc/spec.md
+  -  Waste Management:
+      - Introduction         : WasteManagement/doc/introduction.md
+      - WasteContainerIsle   : WasteManagement/WasteContainerIsle/doc/spec.md
+      - WasteContainerModel  : WasteManagement/WasteContainerModel/doc/spec.md
+      - WasteContainer       : WasteManagement/WasteContainer/doc/spec.md
 
-google_analytics: ['UA-83719923-1', 'fiware-datamodels.readthedocs.io']
+google_analytics: ["UA-83719923-1", "fiware-datamodels.readthedocs.io"]
```

Comment:
is this change really needed?

Reply:
as described in rthd examples. not critical, could be revert